inbox/api/ns_api.py

```diff
@@ -249,6 +251,27 @@ def one_account():
     return encoder.jsonify(g.namespace)
 
 
+@app.route('/account', methods=['DELETE'])
+def account_delete():
+    account = g.namespace.account
+
+    account.disable_sync("account deleted")
+
+    g.db_session.commit()
+
+    try:
+        gevent.sleep(3)
```

Comment:
An ugly spike: waiting for marking account for deletion

Reply:
are you sure that 3 sec will be ok here? may be there is another way to check it?

Reply:
It's enough for now. Later on it must be moved to separate jobs:
- first one is executed on demand (user requests deletion of account)
- second one is executed with some schedule, say, once in an hour (actual deletion of account)

I'll create put a task in backlog.

But for now - I'm sure it's enough